File: 02-217.Contains_Duplicates(E)/Bruteforce.js
// APPROACH Bruteforce  - Time O(N^2), Space O(1) - WONT BE ACCEPTED AS IT HAS N^2 Time

const containsDuplicate = (nums) => {
  for (let i = 0; i < nums.length - 1; i++) {
    for (j = i + 1; j < nums.length; j++) {
      if (nums[i] === nums[j]) return true
    }
  }
  return false
}

const nums = [1, 2, 3, 1]
const result = containsDuplicate(nums)
console.log(result)

File: 02-217.Contains_Duplicates(E)/Bruteforce.py
"""
Input: nums = [1,2,3,1]
Output: true
"""


def contains_duplicates(nums):
    for i in range(len(nums) - 1):
        for j in range(i+1, len(nums)):
            if nums[i] == nums[j]:
                return True
    return False


result = contains_duplicates([1, 2, 3, 4])
print(result)
